### ISSUE TITLE
Keywords force completions to stop

### ISSUE THREAD

**Issue by rchiodo** [microsoft/pylance-release#3968]
I was following this tutorial here (using VS code and Pylance)

https://flask.palletsprojects.com/en/2.2.x/tutorial/factory/

And when attempting to type `app.instance_path`, the intellisense would forcefully close when I typed `app.in`:

![keyword](https://user-images.githubusercontent.com/19672699/219801328-59a14ad9-b76d-476b-9c0f-594226897907.gif)

**Comment by rchiodo** [comment]
Root cause is the error node, This code here thinks there is more than one dot:

```typescript
        const index = ParseTreeUtils.getTokenIndexAtLeft(this._parseResults.tokenizerOutput.tokens, offset) - 1;
        const previousToken = ParseTreeUtils.getTokenAtIndex(this._parseResults.tokenizerOutput.tokens, index);
        if (previousToken?.type === TokenType.Dot || previousToken?.type === TokenType.Ellipsis) {
            // Don't allow multiple dot bring up completions.
            return undefined;
```

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.3.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023311-1-March-2023-PreRelease)
